Analysis/src/QwRootFile.cc

```diff
@@ -1,10 +1,28 @@
+// NOTE: The RNTuple classes are experimental at this point.
+// Functionality, interface, and data format is still subject to changes.
+// Do not use for real data!
+ 
+// Until C++ runtime modules are universally used, we explicitly load the ntuple library.  Otherwise
+// triggering autoloading from the use of templated types would require an exhaustive enumeration
+// of "all" template instances in the LinkDef file.
+R__LOAD_LIBRARY(ROOTNTuple)
```

Comment:
On my linux machine this "macro?" was giving me error. So I removed this and it worked without it.